Assets/VRM/UniVRM/Editor/Tests/UniVRMSerializeTests.cs

```diff
@@ -219,10 +219,32 @@ public void HumanoidTest()
         [Test]
         public void MaterialTest()
         {
-            var model = new glTF_VRM_Material();
+            var model = new glTF_VRM_Material
```

Comment:
Could you add this test case as a new unit tests?

Reply:
Simple case is https://github.com/dwango/UniVRM/pull/189/commits/0eff571b60bb61b670b888446030cd28451470b5#diff-1967e80b396b0aed6d6a20cb02357f56R162

Addition of MaterialTest is extra.

Reply:
I see. It is OK if you can say that the test case previously checked is not affected by that changes in this PR.